manage/deploying/production/heroku_do.rst

```diff
@@ -0,0 +1,226 @@
+=============================
+Deploy your Plone app to Heroku and Digital Ocean
+=============================
+
+.. admonition:: Description
+
+    Since deploying a Plone site might be a challenge for new Plone users, this tutorial aims for solving those difficulties.
+
+
+Introduction
+------------
+
+There are many ways to deploy a Plone site. 
+However, we sometimes would want to deploy our Plone sites for testing and developing Plone before we can find services that fit our needs.
+Therefore, this tutorial will show how to deploy Plone on Heroku and Digital Ocean, which are popular platforms for development.
+
+
+Deploying your Plone site on Heroku
+------------
+
+Heroku is very useful in deploying testing sites since it is free and flexible. 
+It supports many programming languages and necessary addons for different development environments.
+Deploying apps on Heroku is one of the way to show people our work samples without spending money.
+Plone has a `Heroku buildpack <https://plone.org/download>`_ that supports deploying automatically.
+On the other hand, this tutorial will show step-by-step deploying Plone app on Heroku manually for better understanding.
+
+Setting up Heroku account and local environment
+^^^^^^^^^
+
+First of all, to deploy an app on Heroku, we will need an account. 
+Head to `Heroku home page <https://www.heroku.com/>`_ to sign up for a free account and get started.
+
+.. image:: ../images/herokuapp.png
+    :align: center
+    :alt: Heroku sign up page
+
+We then need to install Heroku Toolbelt, which is a tool that helps deploying apps from our local machine to Heroku.
+You can find an installer that suits your computer with full instruction at `Heroku documentation <https://devcenter.heroku.com/articles/heroku-cli>`_.
+
+Download and deploy Plone
+^^^^^^^^^
+
```

Comment:
Not valid reST and wrong header, please follow: https://docs.plone.org/about/rst-styleguide.html